processor/processor.go

```diff
@@ -1167,6 +1167,31 @@ func (proc *HandleT) processJobsForDest(jobList []*jobsdb.JobT, parsedEventList
 			eventsToTransform = eventList
 		}
 
+		// filter out events with unsupported message types
+		filteredEvents := []transformer.TransformerEventT{}
+		for _, event := range eventsToTransform {
+			destinationDef := event.Destination.DestinationDefinition
+			supportedTypes, ok := destinationDef.Config["supportedMessageTypes"]
+			if ok {
+				messageType := event.Message["type"].(string)
+				found := false
+				for _, typ := range supportedTypes.([]interface{}) {
+					if messageType == typ.(string) {
+						found = true
+						filteredEvents = append(filteredEvents, event)
+					}
+				}
+				if !found {
+					pkgLogger.Info("Message Type " + messageType + " not supported for Destination " + destinationDef.Name)
```

Comment:
Would it be better to have it as a Debug Log rather than an Info log as it might be very high in number